Localizations/KeepingYouAwake/zh-Hant-TW.xliff

```diff
@@ -0,0 +1,320 @@
+      </trans-unit>
+      <trans-unit id="PWL-2v-TSS.title">
+        <source>Set Default: 10 minutes</source>
+        <target>10 分鐘（設為預設值）</target>
+        <note>Class = "NSMenuItem"; title = "Set Default: 10 minutes"; ObjectID = "PWL-2v-TSS";</note>
+      </trans-unit>
+      <trans-unit id="T5i-64-xaV.title">
+        <source>2 hours</source>
+        <target>2 小時</target>
+        <note>Class = "NSMenuItem"; title = "2 hours"; ObjectID = "T5i-64-xaV";</note>
+      </trans-unit>
+      <trans-unit id="VeD-dm-81B.title">
+        <source>Set Default: Indefinitely</source>
+        <target>永久（設為預設值）</target>
+        <note>Class = "NSMenuItem"; title = "Set Default: Indefinitely"; ObjectID = "VeD-dm-81B";</note>
+      </trans-unit>
+      <trans-unit id="WWd-z5-2Xu.title">
+        <source>10 minutes</source>
+        <target>10 分鐘</target>
+        <note>Class = "NSMenuItem"; title = "10 minutes"; ObjectID = "WWd-z5-2Xu";</note>
+      </trans-unit>
+      <trans-unit id="YNB-qE-TzE.title">
+        <source>30 minutes</source>
+        <target>30 分鐘</target>
+        <note>Class = "NSMenuItem"; title = "30 minutes"; ObjectID = "YNB-qE-TzE";</note>
+      </trans-unit>
+      <trans-unit id="bCY-Vy-8QT.title">
+        <source>15 minutes</source>
+        <target>15 分鐘</target>
+        <note>Class = "NSMenuItem"; title = "15 minutes"; ObjectID = "bCY-Vy-8QT";</note>
+      </trans-unit>
+      <trans-unit id="fUA-NX-hxF.title">
+        <source>Set Default: 1 hour</source>
+        <target>1 小時（設為預設值）</target>
+        <note>Class = "NSMenuItem"; title = "Set Default: 1 hour"; ObjectID = "fUA-NX-hxF";</note>
+      </trans-unit>
+      <trans-unit id="kO2-hM-cAT.title">
+        <source>5 minutes</source>
+        <target>5 分鐘</target>
+        <note>Class = "NSMenuItem"; title = "5 minutes"; ObjectID = "kO2-hM-cAT";</note>
+      </trans-unit>
+      <trans-unit id="kTz-g8-t11.title">
+        <source>Set Default: 15 minutes</source>
+        <target>15 分鐘（設為預設值）</target>
+        <note>Class = "NSMenuItem"; title = "Set Default: 15 minutes"; ObjectID = "kTz-g8-t11";</note>
+      </trans-unit>
+      <trans-unit id="n3h-ZB-CLV.title">
+        <source>Set Default: 2 hours</source>
+        <target>2 小時（設為預設值）</target>
+        <note>Class = "NSMenuItem"; title = "Set Default: 2 hours"; ObjectID = "n3h-ZB-CLV";</note>
+      </trans-unit>
+      <trans-unit id="pVH-VY-YkW.title">
+        <source>Active</source>
+        <target>啟用</target>
+        <note>Class = "NSMenuItem"; title = "Active"; ObjectID = "pVH-VY-YkW";</note>
+      </trans-unit>
+      <trans-unit id="sft-bA-FTz.title">
+        <source>Set Default: 30 minutes</source>
+        <target>30 分鐘（設為預設值）</target>
+        <note>Class = "NSMenuItem"; title = "Set Default: 30 minutes"; ObjectID = "sft-bA-FTz";</note>
+      </trans-unit>
+      <trans-unit id="tAq-iH-rdw.title">
+        <source>Set Default: 5 hours</source>
+        <target>5 小時（設為預設值）</target>
```

Comment:
Xcode complains that it can't handle this line: `Line 105: PCDATA invalid Char value 16`

I checked the line and replaced the invisible control character after the `5` with a space to make it importable. Would that be an acceptable solution?

Reply:
It's ok, sorry for that.